assets/js/theme/languageSwitch.js

```diff
@@ -0,0 +1,31 @@
+import CTFd from "../index";
+
+export default () => {
+  const form = document.getElementById("language-switch");
+  form.addEventListener("submit", async e => {
+    // Do not open API page
+    e.preventDefault();
+
+    // Set language cookie
+    const data = new FormData(form);
+    const language = data.get("language");
+    document.cookie = `language=${language}`;
```

Comment:
I am unsure if I should add `;SameSite=Lax` to the cookie string. Any help appreciated!

Reply:
It should be Lax or None. I think the first approach with Lax is fine. By default likely it's None but I'd need to see what the browser does.